executor/adapter.go

```diff
@@ -381,6 +381,12 @@ func (a *ExecStmt) logSlowQuery(txnTS uint64, succ bool) {
 		if user != nil {
 			userString = user.String()
 		}
+		if len(tableIDs) > 10 {
```

Comment:
what about `if len(tableIDs) > len("table_ids:")` ?

Reply:
There is a comment for it already.